docs/user-guide/containers.md

```diff
@@ -169,37 +169,263 @@ with the job number.
 
 As we saw above, you can create Singularity images by importing from
 DockerHub or Singularity Hub on ARCHER2 itself. If you wish to create
-your own custom image then you must install Singularity on a system
+your own custom image using Singularity then you must install Singularity on a system
 where you have root (or administrator) privileges - often your own
 laptop or workstation.
 
-We provide links below to instructions on how to install Singularity
-locally and then cover what options you need to include in a Singularity
-recipe file to create images that can run on ARCHER2 and access the
-software development modules. (This can be useful if you want to create
-a custom environment but still want to compile and link against
-libraries that you only have access to on ARCHER2 such as the Intel
-compilers, HPE MPI libraries, etc.)
+There are three different options to install Singularity on your local
+system: install Docker and use the [Docker Singularity image](https://quay.io/repository/singularity/singularity?tab=tags) to build
+Singularity containers, install a virtual machine that you can use to
+build Singularity images, or install Singularity on your local system.
 
-### Installing Singularity on Your Local Machine
+For macOS and Windows users we recommend installing Docker Desktop and
+using the official Singularity image to build your own images. For Linux
+users, we recommend installing Singularity directly on your local system.
 
-You will need Singularity installed on your machine in order to locally
-run, create and modify images. How you install Singularity on your
-laptop/workstation depends on the operating system you are using.
+We cover the mechanism that uses Docker in more detail below for macOS
+and Windows users. If your local system is Linux, you can find
+information on installing Singularity on Linux distribution at:
+
+  - [Installing Singularity on
+    Linux](https://www.sylabs.io/guides/2.6/user-guide/installation.html#install-on-linux)
+
+### Building Singularity images using Docker (macOS/Windows)
+
+!!! tip
+    You should install Docker Desktop to allow you to build Singularity
+    images using Docker. Instructions can be found at:
+     - [Installing Docker Desktop on Windows Professional](https://docs.docker.com/docker-for-windows/install/) 
+     - [Installing Docker Desktop on Windows Home](https://docs.docker.com/docker-for-windows/install-windows-home/) 
+     - [Installing Docker Desktop on macOS](https://docs.docker.com/docker-for-mac/install/)
```

Comment:
This admonition doesn't show bullet points (lines 194-199).

Reply:
Thanks Juan, I must have something wrong with my setup. I'll go ahead and approve Andy's change.